src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -13,17 +13,34 @@
     /** The feedback message to be shown to the user. Contains a description of the execution result */
     public final String feedbackToUser;
 
+    /** This boolean decides whether listing will go with the command */
+    public final boolean hasToList;
```

Comment:
Generally, `has` means composition or aggregation (`hasName`, `hasPersons`). Instead of using `hasTo`, maybe use `should`?